config/deploy/production.rb

```diff
@@ -27,6 +27,8 @@
 server 'lib-orange-prod6', user: 'deploy', roles: %i[web app db]
 
 set :deploy_to, '/opt/rails_app'
+after 'deploy:published', 'sitemap:create'
+after 'deploy:published', 'pulsearch:robots_txt'
```

Comment:
These tasks are not yet defined in `deploy.rb`. Discovered this after deploying to staging and noticing that the sitemap wasn't generated.